integration/default_test.go

```diff
@@ -68,16 +74,21 @@ func testDefault(t *testing.T, context spec.G, it spec.S) {
 			)
 
 			image, logs, err = pack.WithNoColor().Build.
-				WithPullPolicy("never").
-				WithBuildpacks(buildpack, buildPlanBuildpack).
+				WithExtensions(
+					settings.Extensions.UbiNodejsExtension.Online,
+				).
+				WithBuildpacks(
+					settings.Buildpacks.Yarn.Online,
+					settings.Buildpacks.BuildPlan.Online,
+				).
 				WithSBOMOutputDir(sbomDir).
 				WithEnv(map[string]string{"BP_LOG_LEVEL": "DEBUG"}).
+				WithPullPolicy(pullPolicy).
 				Execute(name, source)
 			Expect(err).ToNot(HaveOccurred(), logs.String)
 
 			// Ensure yarn is installed correctly
-
-			container, err = docker.Container.Run.WithCommand("which yarn").Execute(image.ID)
+			container, err = docker.Container.Run.WithCommand("command -v yarn").Execute(image.ID)
```

Comment:
I asked @pacostas about this, we don't install which by default in the ubi stack, this allows the test to run for both ubi and jammy